content/app/development/ux/fields/grouping/repeating/attachments/_index.en.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Attachments in repeating groups
+linktitle: Attachments
+description: Specifics on how to set up attachment uploading in repeating groups
+---
+
+## File attachment in repeating groups
+
+{{%notice warning%}}
+This functionality is not yet integrated into Altinn Studio. Setup must be done manually for now.
+{{%/notice%}}
+
+To set up file uploading components in repeating groups, some additional setup is required.
+
+When uploading file attachments it may be difficult to distinguish which file attachments belongs to which rows in the repeating group,
+and thus which submitted information belongs to each attachment. Therefore, you must set up connections to
+the data model when file uploading are used in repeating groups, so that Altinn can fill out the unique identificator
```

Comment:
identifier?

Reply:
Long form for ID :-) It's the UUID that is generated when an attachment is uploaded. I'm not sure why this appears as changed, I'm pretty sure I just moved these docs..

Reply:
Alright, but is identificator correct still?
<img width="386" alt="image" src="https://user-images.githubusercontent.com/47412359/234880306-4b54dd82-2a75-46bf-848f-c9c76ba2d3cb.png">

Reply:
Oh, sorry! 😆 I read too fast.. You're right! I'll fix it.

Reply:
Interestingly, there are 6 more hits when I search for `identificator` in our docs, but 100+ hits for `identifier` so the last one wins!